src/main/resources/service-commons.base.conf

```diff
@@ -17,28 +17,11 @@ play.crypto.secret = "changeme"
 play.crypto.secret = ${?PLAY_APPLICATION_SECRET}
 
 identity-manager.key.strict = false
+identity-manager.key.strict = ${?IDENTITY_MANAGER_STRICT_PUBKEY_MODE}
 identity-manager.key.public-key = ${?IDENTITY_MANAGER_PUBLIC_KEY}
 identity-manager.key.public-key-provider.type = "url"
 identity-manager.key.public-key-provider.url = "http://keycloak:8080/auth/realms/Renku"
-
-resource-manager.key.strict = false
-resource-manager.key.public-key = ${?RESOURCE_MANAGER_PUBLIC_KEY}
-resource-manager.key.public-key-provider.type = "url"
-resource-manager.key.public-key-provider.url = "http://reverse-proxy/api/resource-manager"
-
-resource-manager.service.host = "http://reverse-proxy/api/resource-manager"
-resource-manager.service.host = ${?RESOURCE_MANAGER_SERVICE_URL}
-
-graph.mutation.service.host = "http://reverse-proxy/api/mutation"
-graph.mutation.service.host = ${?GRAPH_MUTATION_SERVICE_URL}
-
-graph.scope.remote.url = "http://reverse-proxy/api/types"
-graph.scope.remote.url = ${?GRAPH_SCOPE_REMOTE_URL}
-
-authorization.client_id = "renku-services"
-authorization.client_secret = ""
-authorization.client_secret = ${?AUTHORIZATION_CLIENT_SECRET}
-authorization.provider.url = "http://keycloak:8080/auth/realms/Renku/protocol/openid-connect/token"
+identity-manager.key.public-key-provider.url = ${?IDENTITY_MANAGER_INTERNAL_URL}
 
 swagger.authorization.provider.url = "http://localhost/auth/realms/Renku/protocol/openid-connect/auth"
-swagger.authorization.provider.url = ${?RENKU_SERVICES_AUTHORIZATION_URL}
```

Comment:
will require a change here too: https://github.com/SwissDataScienceCenter/renku/blob/development/docker-compose.yml#L176

Reply:
I referenced the corresponding PR